<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>FileSenderWindow</class>
 <widget class="QWidget" name="FileSenderWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Send Files</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../images.qrc" >:/images/cspace32.png</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="status" >
     <property name="text" >
      <string>&lt;b>Status...&lt;/b></string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QListWidget" name="fileList" >
       <property name="selectionMode" >
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="addFilesButton" >
         <property name="text" >
          <string>&amp;Add Files...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeFilesButton" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>&amp;Remove File(s)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeAllButton" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Remove All</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="sendFilesButton" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>&amp;Send Files...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeButton" >
         <property name="text" >
          <string>&amp;Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources>
  <include location="../images.qrc" />
 </resources>
 <connections/>
</ui>
